File: src/Goods.Domain/Regions/FederalRegions.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

public enum FederalRegionsENUM
{
    [Display(Name = "Московская область")]
    Moscow = 1,

    [Display(Name = "Тульская область")]
    Tula = 2,

    [Display(Name = "Краснодарский край")]
    KrasnodarskiyKrai = 3,

    [Display(Name = "Ямало-Ненецкий автономный округ")]
    YamaloNenetskiyAvtonomniyOkrug = 4

}

File: src/Goods.BackOffice/client-app/src/domain/regions/federalRegions.ts
export enum FederalRegions {
	Moscow = 1,
	Tula = 2,
	KrasnodarskiyKrai = 3,
	YamaloNenetskiyAvtonomniyOkrug = 4,
	Kaluga = 5
}

export namespace FederalRegions {
	export const getDisplayName = (fr: FederalRegions): string => {
		switch (fr) {
			case FederalRegions.Moscow:
				return 'Московская область';
			case FederalRegions.Tula:
				return 'Тульская область';
			case FederalRegions.KrasnodarskiyKrai:
				return 'Краснодарский край';
			case FederalRegions.YamaloNenetskiyAvtonomniyOkrug:
				return 'Ямало-Ненецкий автономный округ';
			case FederalRegions.Kaluga:
				return 'Калужская область';
		}
	};
}
